proposals/low-level-struct-improvements.md

```diff
@@ -442,7 +441,7 @@ The [rationale](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-
 
 <a name="rules-unscoped"></a>
 
-To fix this the  language will provide the opposite of the `scoped` lifetime annotation in the syntax `unscoped`.  The keyword `unscoped` will be used to expand the lifetime of a value. It can be applied to any `ref` which is implicitly `unscoped` and has the impact of changing its *ref-safe-to-escape* to *calling method*.
+To fix this the  language will provide the opposite of the `scoped` lifetime annotation by supporting an `UnscopedRefAttribute`.  The attribute can be used to expand the lifetime of a value. It can be applied to any `ref` which is implicitly `scoped` and has the impact of changing its *ref-safe-to-escape* to *calling method*.
```

Comment:
>any `ref` which is implicitly `scoped` [](http://example.com/codeflow?start=212&length=38)

Consider enumerating those implicitly `scoped` `ref` cases if it's easy.

Reply:
Also, is it an error to place `[UnscopedRef]` on some location that doesn't qualify per above?

Reply:
> Consider enumerating those implicitly scoped ref cases if it's easy.

Added a link to the cases described earlier.

> Also, is it an error to place `[UnscopedRef]` on some location that doesn't qualify per above?

Added as an error.